dataset_api/search.py

```diff
@@ -296,13 +302,14 @@ def search(request):
     size = request.GET["size"]
     frm = request.GET["from"]
 
-    if query_string:
+    if query_string != "":
```

Comment:
add both the checks like if query_string and query_string != "":

Reply:
there is an else case.

Reply:
This would throw an exception if query_string is None.